src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
+@import "../fonts/fonts";
+@import "./blocks/nav";
+@import "./blocks/card";
+@import "./blocks/stars";
+@import "./blocks/header";
+@import "./blocks/container";
+
+:root {
+  --hover-color: #00acdc;
+  --card-width: 200px;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto-Medium", sans-serif;
+  --hover-color: #00acdc;
+  --card-width: 200px;
+}
+
+ul {
+  padding-left: 0;
+  margin: 0;
+}
+
+.logo {
+  position: relative;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+}
+
+.is-active {
+  color: var(--hover-color);
+}
+
+.is-active::after {
```

Comment:
Move .is-active::after styles to the nav block file, since it's related to the navigation.

Reply:
still not fixed